### DISCUSSION TITLE
Github actions LSP

### DISCUSSION THREAD

**Discussion by caenrique** [neovim/neovim#24497] [Ideas]
There seems to be an [language server for Github Actions](https://github.com/actions/languageservices/tree/main/languageserver). I haven't tried, and was wondering if someone has successfully used it.
Would be awesome to include it as a config here if possible

**Comment by clason** [comment]
Note that Neovim does not provide configs for language servers out of the box; you may be looking for https://github.com/neovim/nvim-lspconfig (which is a distinct project).

**Reply by caenrique** [reply]
My fault 😓 I followed a link from https://github.com/neovim/nvim-lspconfig/tree/master#contributions thinking that it was the discussions page of that repo, but instead it links to here, which is a bit misleading.

**Reply by clason** [reply]
Ah, thanks for the heads-up! I see now how that happened. Not sure I like it, but unclear how to best handle it.

**Comment by nyngwang** [comment]
This is cool, thanks for the info!
